processor/processor.go

```diff
@@ -1559,7 +1593,7 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 		}
 		statusList = append(statusList, &newStatus)
 
-		source, err := proc.getSourceBySourceID(sourceId)
+		source, err := proc.getSourceBySourceID(sourceID)
 		if err != nil {
 			continue
```

Comment:
This looks like another case where we could set the span status to error, right?